definitions/npm/pg_v7.x.x/flow_v0.28.x-/pg_v7.x.x.js

```diff
@@ -212,12 +212,12 @@ declare module pg {
    * ToDo: should find a better way.
   */
   declare class Query extends Promise<ResultSet> {
-    then<U>( onFulfill?: (value: ResultSet) => Promise<U> | U,
-      onReject?: (error: PG_ERROR) => Promise<U> | U
+    then<U>( null | onFulfill?: (value: ResultSet) => Promise<U> | U,
```

Comment:
`onFulfill?` means you can just call `then()`, without the need to pass that option in. The `?` in ` null | onFulfill?` isn't valid syntax.